api/v1alpha1/drplacementcontrol_types.go

```diff
@@ -92,6 +92,7 @@ const (
 	ProgressionUpdatedPlacement                    = ProgressionStatus("UpdatedPlacement")
 	ProgressionEnsuringVolSyncSetup                = ProgressionStatus("EnsuringVolSyncSetup")
 	ProgressionSettingupVolsyncDest                = ProgressionStatus("SettingUpVolSyncDest")
+	ProgressionSetDeleting                         = ProgressionStatus("Deleting")
```

Comment:
Why is it `ProgressionSetDeleting`? Why not `ProgressionDeleting` like the others? What does `Set` mean here?